utopia-remix/app/routes/internal.projects.$id.access.tsx

```diff
@@ -0,0 +1,35 @@
+import { ActionFunctionArgs } from '@remix-run/node'
+import { Params } from '@remix-run/react'
+import { ensure, handle, requireUser } from '../util/api.server'
+import { Status } from '../util/statusCodes'
+import { setProjectAccess } from '../models/projectAccess.server'
+import { asNumber } from '../util/common'
+import { AccessLevel, asAccessLevel } from '../types'
+import { getProject } from '../models/project.server'
+
+export async function action(args: ActionFunctionArgs) {
+  return handle(args, { POST: handleChangeProjectAccess })
+}
+
+export async function handleChangeProjectAccess(req: Request, params: Params<string>) {
+  const user = await requireUser(req)
+  const { id } = params
+  ensure(id != null, 'id is null', Status.BAD_REQUEST)
+
+  const project = await getProject({ id: id, owner_id: user.user_id })
```

Comment:
same here, we don't need the whole project, we just want to know if the user owns that project, which can be done with a dedicated function that returns the `owner_id` field from the DB

Reply:
I don't think this is resolved, could you have a look at updating it please?

Reply:
removed (the validator checks it now)